scripts/migrate-attachments.pl

```diff
@@ -59,33 +68,68 @@
 
 my $dbh = Bugzilla->dbh;
 
+if ($options{migrate}) {
```

Comment:
Do we want an option to only migrate attachments above a certain size?

Reply:
This should technically happen already when migrating from DB to S3. If the 'attachment_s3_minsize' parameter is set, the S3 module will default to storing that data in the DB if less that the minsize value.

When migrating from S3 to DB, all data will be migrated regardless of size which I expect people will be fine with from the most part.

And DB to filesystem and filesystem to DB are basically no performance change from one to the other.

Reply:
Aha, you're right we're calling the S3 module any which does that. Awesome!